packages/table/src/utils/types.ts

```diff
@@ -0,0 +1,51 @@
+import { Primitive, Threshold } from '@synchro-charts/core';
+import { ErrorDetails } from '@iot-app-kit/core';
+
+export type ItemRef = {
+  $cellRef: {
+    id: string;
+    resolution: number;
+  };
+};
+
+export type Item = {
+  [key in string]: ItemRef | unknown;
+};
+
+export type CellItemProps = {
+  value?: Primitive;
+  error?: ErrorDetails;
+  isLoading?: boolean;
+  threshold?: Threshold;
+};
+
+export class CellItem {
```

Comment:
A class is not a type, unsure if this is the right place for this. is there a reason to use a class for this?